src/metadata.ts

```diff
@@ -0,0 +1,92 @@
+import { getInput } from '@actions/core';
+import { context } from '@actions/github';
+import MetadataController from 'issue-metadata';
```

Comment:
You have to add `issue-metadata` to the list of dependencies - `yarn add issue-metadata`